src/ch01-01-installation.md

```diff
@@ -1,128 +1,279 @@
+read it offline. Run `rustup doc` to open the local documentation in your
+browser.
+
+Any time a type or function is provided by the standard library and you’re not
+sure what it does or how to use it, use the application programming interface
+(API) documentation to find out!
+ -->
+
+### Solução de Problemas
+
+Para checar se você tem Rust instalado corretamente, abra um terminal e digite
+esta linha:
+
+```text
+$ rustc --version
+```
+
+Você deverá ver o número, o hash e a data do commit da última versão estável
+lançada no seguinte formato:
 
-Se não aparecer a mensagem acima e você está em um ambiente Windows, verifique
-se o Rust aparece na variável `%PATH%` do seu sistema.
+```text
+rustc x.y.z (abcabcabc yyyy-mm-dd)
+```
 
-Se mesmo assim não funcionar, existem vários lugares onde você pode pedir ajuda.
-A maneira mais fácil é pedir ajuda no
-[canal IRC #rust do irc.mozilla.org][irc]<!-- ignore -->, que você pode acessar
-via [Mibbit][mibbit]. Você vai falar com vários outros _Rustaceans_ (um apelido
-bobo que usamos entre nós) que podem ajudá-lo em suas dúvidas. Você também pode
-buscar ajuda no [fórum do Rust][users] e no [Stack Overflow][stackoverflow].
+Se aparecer a mensagem acima, Rust foi instalado com sucesso! Se você não ver
+esta informação e você estiver no Windows, cheque que Rust está no sua variável
+`%PATH%` de ambiente. Se tudo isso estiver correto e Rust ainda não funcionar,
+existem vários locais em que você pode pedir ajuda. O mais fácil é o canal
+#beginners no [Discord oficial de Rust][discord]. Lá, você pode falar com
+outros Rustáceos (um apelido bobo que damos a nós mesmos) que podem te ajudar.
+Outros grandes recursos incluem [o fórum de usários][users] e o [Stack
+Overflow][stackoverflow].
 
-[irc]: irc://irc.mozilla.org/#rust
-[mibbit]: http://chat.mibbit.com/?server=irc.mozilla.org&channel=%23rust
+[discord]: https://discord.gg/rust-lang
 [users]: https://users.rust-lang.org/
 [stackoverflow]: http://stackoverflow.com/questions/tagged/rust
 
+### Local Documentation
+
+The installer also includes a copy of the documentation locally, so you can
+read it offline. Run `rustup doc` to open the local documentation in your
+browser.
+
+Any time a type or function is provided by the standard library and you’re not
+sure what it does or how to use it, use the application programming interface
+(API) documentation to find out!
+
 ### Documentação local
 
 O instalador também inclui uma cópia local da documentação para que você possa
-acessá-la _offline_. Execute o comando `rustup doc` para abrir a documentação
-local no seu navegador.
+acessá-la offline. Execute o comando `rustup doc` para abrir a documentação no
+seu navegador.
 
-Sempre que você se deparar com um tipo ou função fornecido pela biblioteca
-padrão que você não tem certeza do que ele faz, use a documentação da API para
+Sempre que um tipo ou for fornecido pela biblioteca-padrão e você não tiver
```

Comment:
... um tipo ou **uma função** for ...

E tem hífen em `biblioteca-padrão`? :thinking:

Reply:
Lembro de ver em muitos lugares antes mas agora pesquisando vi muitos lugares usando sem :thinking: Exemplo: https://pt.wikipedia.org/wiki/Biblioteca_padr%C3%A3o_do_C

Se é do c eu não discuto!